posts/models.py

```diff
@@ -6,3 +6,4 @@
 class Post(models.Model):
     content = models.CharField(max_length=256)
     author = models.ForeignKey(User)
+    created_at = models.DateTimeField(auto_now_add=True)
```

Comment:
We decided that posts should not be edited, so the `updated_at` field would be useless.